timur/lib/client/jsx/components/model_map/tree_layout.jsx

```diff
@@ -87,6 +87,8 @@ class TreeLayout {
 
     let grid = [];
 
+    this.grid = grid;
```

Comment:
Do you still need this?

Reply:
Yeah has to be retained to figure out how many siblings each node has.